src/adhocracy_core/adhocracy_core/sheets/principal.py

```diff
@@ -436,6 +436,26 @@ class ServiceKontoSchema(MappingSchema):
 )
 
 
+class IServiceKontoSettings(ISheet):
+    """Marker interface for the ServiceKonto settings sheet."""
+
+
+class ServiceKontoSettingsSchema(MappingSchema):
+    """Data structure for public ServiceKonto user information."""
+
+    enabled = Boolean()
```

Comment:
I find your choice of var names - `Settings` and `enabled` - interesting. Will there be any settings besides the current boolean that we'll deal with?